src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/index.jsx

```diff
@@ -14,37 +14,42 @@ import getEmbedUrl from '#lib/utilities/getEmbedUrl';
 
 const staticAssetsPath = `${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN}${process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH}`;
 
-const liveRadioPlaceholderImageSrc = `${staticAssetsPath}images/amp_audio_placeholder.png`;
+const audioPlaceholderImageSrc = `${staticAssetsPath}images/amp_audio_placeholder.png`;
 
-const MediaPlayerOuterWrapper = styled.div`
+const getMediaInfo = assetId => ({
+  title: assetId === 'liveradio' ? 'Live radio' : 'On-demand radio',
```

Comment:
Can we rename the titles (or is this format required for SMP?) `Live Radio` / `On Demand Radio`

Reply:
`title` here becomes the title of the outer iframe - there's probably some a11y requirement around what this should be

Reply:
@greenc05 are you able to advise on what the title attribute of the audio player embed iframe should be? Thanks